### PR TITLE
Add field link for field component

### PR DESCRIPTION
**Pull request by beveryday**

### CODE REVIEW

File: Field/index.jsx [hunk 1]
```diff
@@ -26,7 +27,8 @@ const classes = {
   iconInput: `${baseClass}--icon__input`,
   iconLabel: `${baseClass}--icon__label`,
   input: `${baseClass}__input`,
-  label: `${baseClass}__label`
+  label: `${baseClass}__label`,
+  skipLink: `${baseClass}--skip__link`
```
Comment (Deschtex) [inline Field/index.jsx]: Is this class being used?

File: Field/index.jsx [hunk 2]
```diff
@@ -225,7 +236,8 @@ const Field = React.createClass({
     const ids = id
       ? {
         input: `${id}__input`,
-        label: `${id}__label`
+        label: `${id}__label`,
+        link: `${id}__${fieldLink}__fieldLink`
```
Comment (Deschtex) [inline Field/index.jsx]: I believe it should be written as `field-link` instead of `fieldLink`. WDYT?

File: FieldLink/index.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import classNamesBind from 'classnames/bind'
+import defaultStyles from './styles.scss'
+
+const baseClass = 'field-link'
+
+class FieldLink extends React.Component {
+  propTypes: {
+    id: PropTypes.string,
+    label: PropTypes.string.isRequired,
+    onFieldLinkClick: PropTypes.func
```
Comment (deepsweet) [inline FieldLink/index.jsx]: `isRequired`?

File: Field/index.jsx (hunk 2 above)
Comment (beveryday) [inline Field/index.jsx]: Sounds good to me. I wasn't sure

File: FieldLink/index.jsx (hunk 3 above)
Comment (beveryday) [inline FieldLink/index.jsx]: Ya I think we can remove that

**Pull request merged**